pkg/resources/prometheusRules_test.go

```diff
@@ -181,7 +181,7 @@ func TestReconcileAlerts(t *testing.T) {
 			ExistingRules: []*monv1.PrometheusRule{},
 			Installation: &integreatlyv1alpha1.RHMI{
 				ObjectMeta: v1.ObjectMeta{
-					DeletionTimestamp: now(),
+					//	DeletionTimestamp: now(),
```

Comment:
If it's not required, please remove instead of commenting out.

Reply:
done

Reply:
it's not done

Reply:
sorry, updated. Thank you